<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseHobby</class>
 <widget class="QDialog" name="chooseHobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择你的爱好标签（最多两项）</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_sports">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>67</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>运动</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_boardgame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>桌游</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_eatout">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>聚餐</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_trip">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>出游</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_others">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>其他</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
